### DISCUSSION TITLE
Inserting IEnumerable

### DISCUSSION THREAD

**Discussion by asherber** [ClosedXML/ClosedXML#2362] [Q&A]
I recently upgraded from v0.92 to v0.102. Per the [docs](https://github.com/ClosedXML/ClosedXML/wiki/Copying-IEnumerable-Collections), I used to be able to add an `IEnumerable` to a sheet by doing `ws.Cell(1,1).Value = myEnumerable`. But because of the recent change to `IXLCell.Value` detailed [here](https://github.com/ClosedXML/ClosedXML/releases/tag/0.100.0#:~:text=Cell%20value%20is%20now%20strongly%20typed), this no longer works.

1. Is the proper way to add an `IEnumerable` by using `ws.Cell(1,1).InsertData(myEnumerable)`, and does this give the same results as the earlier apporach?
2. If so, should the docs be updated to reflect this?

**Comment by jahav** [comment]
Ad 1: Yes, the proper way is to use InsertData. It produces the same result as the original, see  the `SetEnumerable` method in original setter https://github.com/ClosedXML/ClosedXML/blob/0.97.0/ClosedXML/Excel/Cells/XLCell.cs#L670

Ad 2: I have added obsolete notice to the wiki page. It links to up-to-date docs and a migration guide noticing the change.
